website/docs/feature-flag-tutorials/sveltekit/feature-flags-sveltekit.md

```diff
@@ -0,0 +1,267 @@
+        items.push({ id: items.length + 1, name: formData.get('name'), completedDays: [] });
+        return items;
+      });
+
+      // reset the form
+      form.reset();
+    }
+  }
+</script>
+
+<dialog open={habitsFull}>
+  <h2>❌ Maximum Habits Reached</h2>
+  <p>You can only have up to {maxHabits} on the free tier. Purchase a premium version to unlock more.</p>
+  <form method="dialog">
+    <button>OK</button>
+  </form>
+</dialog>
+
+<form on:submit|preventDefault={addHabit}>
+  <input type="text" name="name" />
+  <button type="submit"> Add new habit </button>
+</form>
+```
+
+What's happening here? A few things:
+
+- An input and a button to add new habits to the store, until an arbitrary limit is reached
+- A `maxHabits` prop is used to determine that limit
+- When this maximum limit is reached, a modal dialog opens
+- We reset the form after submission to clear the input
+
+<video
+  width="600px"
+  preload="metadata"
+  autoplay
+  loop
+  title="A maximum number of habits on the free tier"
+  src="/media/habits.mp4"
+  controls> </video>
+
+## Showing a different component based on the feature flag
+
+On to the main topic, adding feature flags.
+
+Go to your Unleash dashboard, and create new project (you're welcome to use the default project here).
+![alt text](./proj.png)
+
+Then create a feature flag called `maxHabitsIncreased`
+
+![alt text](./feat.png)
+
+Bbased on whether this flag is enabled or not, we'll set the `maxHabits` value to either 6 or 2. You could set this directly in a flag value if you wanted as well.
+
+### Basic toggle
+
+We'll use the official [Unleash Svelte SDK](https://docs.getunleash.io/reference/sdks/svelte). We'll wrap our `App.svelte` with a context provider exported from the SDK, like so:
+
+```svelte
+<script>
+  // src/routes/+page.svelte
+  import App from '../lib/App.svelte';
+  import { FlagProvider } from '@unleash/proxy-client-svelte';
+
+  const config = {
+    url: 'https://eu.app.unleash-hosted.com/jdfkdjfkd/api/frontend', // Your Front-end API
+    clientKey: '', // Front-end API token (or proxy client key)
+    appName: 'habits'
+  };
+</script>
+
+<FlagProvider {config}>
+  <App />
+</FlagProvider>
+```
+
+Note that I’m using the URL and API key directly in the code right now, but you’d want to put these in an env file.
+
+Now that our SDK is setup, we can modify our `App.svelte` to set the value of the variable based on the feature flag.
+
+```diff
++  import { useFlag } from '@unleash/proxy-client-svelte';
++  const maxHabitsIncreased = useFlag('maxHabitsIncreased');
++  let maxHabits = $maxHabitsIncreased ? 6 : 2;
+-  lex maxHabits = 3;
+```
+
+## Conclusion
+
+That's it, you have a Sveltekit app with fleature flags. More precisely, you've learned:
+
+- How to make a habit tracking app with Sveltekit
+- How to add feature flags and how to add them to a fullstack app
+- The different approaches to feature flagging on a static vs SSR context
+
+Enjoy, see you soon.
```

Comment:
I'd suggest removing this line for now and maybe we can replace it with pointing devs to a link to other useful docs in the future. Example:

Take a look at our [insert link] documentation for a deeper dive into advanced use cases with Sveltekit.